.gitlab-ci.yml

```diff
@@ -137,6 +137,7 @@ include:
   - local: 'benchmarks/timing/config.yml'
   - local: 'benchmarks/b0_tracker/config.yml'
   - local: 'benchmarks/others/config.yml'
+  - local: 'benchmarks/backwards_ecal/config.yml
```

Comment:
```suggestion
  - local: 'benchmarks/backwards_ecal/config.yml'
```
Missing a `'`

Reply:
Ah, well spotted. I've fixed that now. Cheers!